climpred/prediction.py

```diff
@@ -279,14 +280,14 @@ def predictability_horizon(skill,
     """
```

Comment:
we dont have tests for predictability horizon for now. I havent used this in weeks. can be refactored but not in this PR.

Reply:
I think this is a v2 issue (https://github.com/bradyrx/climpred/issues/46). We still need to go back through it and make it more robust. As it currently stands, it was hacked together for our needs.